media/js/products/vpn/ppa-experiment.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+if (/\sFirefox/.test(navigator.userAgent)) {
+    const links = document.getElementsByClassName('js-fxa-product-button');
+    Array.from(links).forEach((link) => {
+        'use strict';
+
+        link.addEventListener('click', () => {
+            navigator.privateAttribution.measureConversion({
+                task: 'task_id', // Unique task ID. Supplied by Mozilla.
```

Comment:
Has this been provided yet? I assume our unique task ID is not actually `task_id`.

Reply:
Both `task` and `histogramSize` have not been provided yet. I will update the PR once they are available